r4/datatype_reference.bal

```diff
@@ -78,3 +78,54 @@ public type Reference record {|
     Identifier identifier?;
     string display?;
 |};
+
+# Create a relative Reference to FHIR Resource.
+#
+# + referenceResourceModel - FHIR Resource model of the Reference
+# + relativePathToReference - Relative path to Reference
+# + return - FHIRError if the Reference resource type is not defined, else FHIR Reference
+public isolated function createRelativeFhirReference(typedesc<anydata> referenceResourceModel, string relativePathToReference) returns Reference|FHIRError {
+    return createNonContainedFhirReference(referenceResourceModel, relativePathToReference);
+}
+
+# Create an absolute Reference to FHIR Resource.
+#
+# + referenceResourceModel - FHIR Resource model of the Reference
+# + absolutePathToReference - Absolute path to Reference
+# + return - FHIRError if the Reference resource type is not defined, else FHIR Reference
+public isolated function createAbsoluteFhirReference(typedesc<anydata> referenceResourceModel, string absolutePathToReference) returns Reference|FHIRError {
+    return createNonContainedFhirReference(referenceResourceModel, absolutePathToReference);
+}
+
+# Create a contained Reference to FHIR Resource.
+#
+# + targetResource - FHIR Resource which the Reference is to be contained/embedded
+# + resourceTypeReference - Resource typed Reference to be contained/embedded
+# + return - FHIRError if the target FHIR Resource is not a valid FHIR resource, else FHIR Reference
+public isolated function createContainedFhirReference(map<anydata> targetResource, Resource resourceTypeReference) returns Reference|FHIRError {
+    string resourceType = resourceTypeReference.resourceType;
+    anydata? targetContained = targetResource["contained"];
```

Comment:
Use a constant for `contained`

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/99494063ef9207cb20a4da3b98472749f619a5ea